src/App.js

```diff
@@ -1,11 +1,25 @@
+import React, { PureComponent } from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './component/Calculator';
+import Quote from './component/Quote';
+import Home from './component/Home';
+import Headers from './component/Header';
+import Footer from './component/Footer';
 
-function App() {
-  return (
-    <>
-      <Calculator />
-    </>
-  );
+class App extends PureComponent {
+  render() {
+    return (
+      <>
+        <Headers />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calDesign" element={<Calculator />} />
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` " , please kindly ensure the general layout of your calculator page matches the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) both in content and alignment. Your page is missing the "  `Let's do the math!` " text, please kindly add it and make sure it is left align.  The calculator itself occupies about 2/3 of the page and it is right aligned, please kindly fix that too.